nss-evening-cohort-14/gitsub-team-no-name
# Overview styling

## Description
- Making form background black
- Adding card borders
- Adding horizontal lines

## Related Issue
https://github.com/nss-evening-cohort-14/gitsub-team-no-name/issues/23

## Motivation and Context
Styling changes needed for consistency across pages.

## How Can This Be Tested?
Pull down the branch, check that repository cards have a border, form input background is black.

## Screenshots (if appropriate):
N/A

## Types of changes

- [X ] New feature (non-breaking change which adds functionality)
